components/calendar/src/buddhist.rs

```diff
@@ -55,8 +55,11 @@ const BUDDHIST_ERA_OFFSET: i32 = 543;
 ///
 /// # Era codes
 ///
-/// This calendar supports one era, `"be"`, with 1 B.E. being 543 BCE
-
+/// This calendar supports one era, `"be"`, with 1 B.E. being 543 BCE.
```

Comment:
Question: is there an issue tracking full era code compliance (where we accept both the canonical era and the aliases)?

Reply:
I'm imagining that will be the final state of #3869